.github/workflows/rollback.yml

```
@@ -0,0 +1,99 @@
+    inputs:
+      image_tag:
+        description: '롤백할 이미지 태그 (예: develop-abc1234)'
+        required: true
+        type: string
+
+jobs:
+  rollback:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)